openstack/nova/templates/etc/_nova.conf.tpl

```diff
@@ -23,7 +23,7 @@ default_schedule_zone = {{.Values.global.default_availability_zone}}
 default_availability_zone = {{.Values.global.default_availability_zone}}
 
 rpc_response_timeout = {{ .Values.rpc_response_timeout | default .Values.global.rpc_response_timeout | default 60 }}
-rpc_workers = {{ .Values.rpc_workers | default .Values.global.rpc_workers | default 1 }}
+rpc_workers = {{ .Values.rpc_workers }}
```

Comment:
I can only find that setting for in `neutron.conf` - I don't think we need that setting, as it's unused in Nova.

Reply:
Yes, I didn't find it in the source either. I've removed it.